File: Day 5/Homework/Passing Data Property ObjC/Passing Data Property ObjC/ViewController.m
//
//  ViewController.m
//  Passing Data Property ObjC
//
//  Created by Consultant on 8/24/22.
//

#import "ViewController.h"
#import "ViewController2.h"

@interface ViewController ()

@end

@implementation ViewController

//@IBAction func onButtonTap() {
//        let vc = storyboard?.instantiateViewController(withIdentifier: "vc2") as! ViewController2
//        vc.passText = "Hello World"
//
//        present(vc, animated: true, completion: nil)
//    }

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (IBAction)onButtonTap:(id)sender {
    ViewController2 *vc2 = [self.storyboard instantiateViewControllerWithIdentifier:@"vc2"];
    vc2.passText = @"Hello World";
    //To call a func need self in brackets
    [self presentViewController:vc2 animated:YES completion:nil];
}

@end

File: Day 5/Tab Example Class/Tableview Class/ViewController.swift
//
//  ViewController.swift
//  Tableview Class
//
//  Created by Consultant on 8/24/22.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    @IBAction func myButton(_ sender: Any) {
        performSegue(withIdentifier: "mySegue", sender: nil)
    }
    
}

